src/include/brain/brain_defs.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+namespace terrier::brain {
+
+enum class ExecutionOperatingUnitType : uint32_t {
```

Comment:
Is there an enum of operations we can leverage instead of possibility duplicating, or is this distinct?

Reply:
This is distinct. We've discussed with Prashanth about this. These are similar to the operations in the execution engine but different.